mkt/settings.py

```diff
@@ -1001,8 +1001,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
```

Comment:
`REDIS_BACKENDS` is also defined in `sites/*`, and they use `private` settings. I assume thos are not using the dict format either and will fail with the update...